snap/bin/security-start.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+export KONG_SNAP="${SNAP}/bin/kong-wrapper.sh"
```

Comment:
Do you need quotes here when you've already used {}s?

Reply:
SC2086 from shellcheck says so